dags/iop/parser.py

```diff
@@ -51,6 +55,18 @@ def __init__(self) -> None:
                 extraction_function=self._extract_authors,
                 required=True,
             ),
+            TextExtractor(
+                destination="title",
+                required=False,
+                source="front/article-meta/title-group/article-title",
+                extra_function=lambda x: " ".join(x.split()),
+            ),
+            TextExtractor(
+                destination="subtitle",
+                required=False,
+                source="front/article-meta/subtitle",
+                extra_function=lambda x: " ".join(x.split()),
```

Comment:
Wanted to remove all double spaces, new lines and etc.

Reply:
do we have such cases? why we need to handle that?